src/lib/stores/tokens/tokens.store.ts

```diff
@@ -0,0 +1,211 @@
+import uniswapTokenList from '@uniswap/default-token-list';
+import type { TokenInfo } from '@uniswap/token-lists';
+import { get, writable } from 'svelte/store';
+import * as storedTokens from './stored-custom-tokens';
+import assert from '$lib/utils/assert';
+import { Utils } from 'radicle-drips';
+
+interface DefaultTokenInfoWrapper {
+  info: TokenInfo;
+  source: 'default';
+}
+
+export interface CustomTokenInfoWrapper {
```

Comment:
Since this is also used in the custom token store, I might consider extracting it out into a separate file.

Reply:
👍 If you don't mind, I'd do this when / if we add at least one or two more types — only reason I didn't already is that I thought having a types file and needing to export even the private types wasn't worth it for only 1 public definition.